polaris-core/src/main/java/org/apache/polaris/core/persistence/cache/EntityCache.java

```diff
@@ -76,15 +81,22 @@ public EntityCache(@Nonnull PolarisMetaStoreManager polarisMetaStoreManager) {
         };
 
     long weigherTarget =
-        PolarisConfiguration.loadConfig(FeatureConfiguration.ENTITY_CACHE_WEIGHER_TARGET);
+        polarisCallContext
```

Comment:
For cases like this, either  the configuration store needs non-realm-specific config APIs or we should construct an EntityCacheConfig object at startup that users can set using the application.properties file.

Reply:
> the configuration store needs non-realm-specific config APIs 

Agreed. Notably, this doesn't work as written on main today. So this isn't a new problem introduced by the PR.

Having said that, we should fix this and I would be okay fixing this in this PR. Do you have a way you'd like to model this?